### PR TITLE
Local Swarm that uses Aws databases

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: localDev/localSwarmAwsDb/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Running Application in local docker swarm (with aws databases)
+
+The scripts and configuration in this folder will allow you to bring up the full BristechSRM system as a local docker swarm, and access frontend from windows on `http://localhost:8080`.
+The Api gateway will also be avaiable on `localhost:8081` and the auth service on `localhost:9003`.
+
+## Install Requirements 
+You will require git bash or an equivalent installed, as well as Docker. On windows 7, we have used docker toolbox : https://www.docker.com/products/docker-toolbox. 
+Note for git bash, you either need to use the executable "C:\Program Files\Git\git-bash.exe", or call bash or sh (found in "C:\Program Files\Git\bin\") with the -l (--login) flag.
```
Comment (pallenSL) [inline localDev/localSwarmAwsDb/README.md]: Not true any more, I hope(?)

File: localDev/localSwarmAwsDb/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ - The secrets config file for Auth must be placed under ./configs/auth/ as secrets.Auth.config. 
+    - Note, the certificateFile setting value must be of the format "/certFileName.pfx" (There is an issue around the location of the file, so it will be place in the file system root for the service)
+ - The corresponding certificate file must be placed under ./configs/auth/ . E.g. if the certificateFile specfied is "/certFileName.pfx", the file needs to be "certFileName.pfx".
+   A template is ready at ./configs/comms/template.secrets.Auth.config
+
+### Comms
+ - The secrets config file for Comms must be placed under ./configs/comms/ as secrets.Comms.config. 
+  A template is ready at ./configs/comms/template.secrets.Comms.config
+
+### TrelloMigrate
```
Comment (pallenSL) [inline localDev/localSwarmAwsDb/README.md]: No such directory.  Should there be?

File: localDev/localSwarmAwsDb/destroy.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+docker-machine rm manager0 -y
+docker-machine rm worker0 -y
+docker-machine rm worker1 -y
```
Comment (pallenSL) [inline localDev/localSwarmAwsDb/destroy.sh]: POSIX, please.

File: localDev/localSwarmAwsDb/README.md (hunk 2 above)
Comment (jebbinSCL) [inline localDev/localSwarmAwsDb/README.md]: Ah thanks

File: localDev/localSwarmAwsDb/setupAll.sh [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+"C:\Program Files\Oracle\VirtualBox\VBoxManage.exe" controlvm "manager0" natpf1 "frontend,tcp,127.0.0.1,8080,,8080"
+"C:\Program Files\Oracle\VirtualBox\VBoxManage.exe" controlvm "manager0" natpf1 "gateway,tcp,127.0.0.1,8081,,8081"
+"C:\Program Files\Oracle\VirtualBox\VBoxManage.exe" controlvm "manager0" natpf1 "auth,tcp,127.0.0.1,9003,,9003"
+
+docker-machine scp setupServices.sh manager0:~/setupServices.sh
+
+echo "SSHing into manager0 and bringing up services"
+docker-machine ssh manager0 "sleep 30; ./setupServices.sh"
+
+eval $(docker-machine env --shell bash manager0)
```
Comment (pallenSL) [inline localDev/localSwarmAwsDb/setupAll.sh]: It's a warning for a reason.

File: localDev/localSwarmAwsDb/destroy.sh (hunk 3 above)
Comment (jebbinSCL) [inline localDev/localSwarmAwsDb/destroy.sh]: What do you mean? The line ending?

Comment (pallenSL) [inline localDev/localSwarmAwsDb/destroy.sh]: :yes;  The first attempt at standardizing all the disparate Unices was called "POSIX", it says you have a newline last.
It's from back when file transfers weren't that reliable...

File: localDev/localSwarmAwsDb/setupAll.sh (hunk 4 above)
Comment (pallenSL) [inline localDev/localSwarmAwsDb/setupAll.sh]: Why does it do this at the end anyway?  It won't do anything.

Comment (jebbinSCL) [inline localDev/localSwarmAwsDb/setupAll.sh]: removed

**Comment by pallenSL** [comment]
:shipit:

**Pull request merged**